qa/tasks/cephfs/kernel_mount.py

```diff
@@ -223,6 +237,39 @@ def read_debug_file(self, filename):
                 return None
             raise
 
+    def _dynamic_debug_control(self, enable):
+        """
+        Write to dynamic debug control file.
+        """
+        if enable:
+            fdata = "module ceph +p"
+        else:
+            fdata = "module ceph -p"
+
+        pyscript = dedent("""
+            import os
+
+            dyfile = "/sys/kernel/debug/dynamic_debug/control"
+            print(open(dyfile).write("{fdata}"))
+            """).format(fdata=fdata)
+
+        self.client_remote.sh(['sudo', 'modprobe', 'ceph'],
+                              timeout=(5*60))
+        self.client_remote.sh(['sudo', 'python3', '-c', pyscript],
+                              timeout=(5*60))
```

Comment:
```suggestion
        if enable:
            fdata = "module ceph +p"
        else:
            fdata = "module ceph -p"

        self.run_shell_payload(f"""
sudo modprobe ceph
echo '{fdata}' | sudo tee /sys/kernel/debug/dynamic_debug/control
""")
```

Reply:
(I prefer to avoid Python when possible.)

Reply:
Okay, will fix it.